File: C_Programming/Competitive_Programming/Assignment12/program4.c
//Write a program which accept number from user and count frequency of 4 in it

// Input : 2395
// Output : 0

// Input : 101844
// Output : 2                    

#include<stdio.h>

int CountTwo(int iNo)
{
    int iDigit = 0;
    int iCnt = 0;

    while(iNo > 0)
    {
        iDigit = iNo % 10;
        if(iDigit == 4)
        {
            iCnt++;
        }
        iNo = iNo / 10;
    }

    return iCnt;
}

int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter number : ");
    scanf("%d",&iValue);

    iRet = CountTwo(iValue);

    printf("Total Count of 4 in Number : %d",iRet);
    
    return 0;
}

File: Java_Programming/Competitive_Programming/Assignment21/program4.java
// Write a program to count total number of factor of a given number 

class logic 
{
    void displayFactor(int iNo)
    {
        int iCnt = 0;
        int iFact = 0;

        for(iCnt = 1; iCnt < iNo; iCnt++ ) 
        { 
           if(iNo % iCnt == 0)
           {
                iFact++;
           }
        }

        System.out.println("Count of Factors : "+iFact);
    }
}

class program4
{
    public static void main(String A[] ) 
    {
        logic obj = new logic();
        obj.displayFactor(20);
    }
}
